screens/Home.js

```diff
@@ -93,67 +107,55 @@ class Home extends React.Component {
       }
     } else if (prevProps.statusBar !== statusBar) {
       this.changeStatusBar(statusBar);
-    } else if (prevProps.statistics !==statistics) {
+    } else if (prevProps.statistics !== statistics) {
       AnalyticsManager.getInstance().allowTracking(statistics);
     } else if (prevProps.reminders !== reminders) {
       NotificationsManager.getInstance().checkPermissions(reminders);
-      NotificationsManager.getInstance().updateReminders(
-        reminders,
-        reminderSound,
-        reminderBanis
-      );
+      NotificationsManager.getInstance().updateReminders(reminders, reminderSound, reminderBanis);
     } else if (prevProps.reminderSound !== reminderSound) {
-      NotificationsManager.getInstance().updateReminders(
-        reminders,
-        reminderSound,
-        reminderBanis
-      );
+      NotificationsManager.getInstance().updateReminders(reminders, reminderSound, reminderBanis);
 
       if (actions.REMINDER_SOUNDS.indexOf(reminderSound) !== 0) {
-        const sound = new Sound(
-          reminderSound,
-          Sound.MAIN_BUNDLE,
-          (error) => {
-            if (error) {
-              return;
-            }
-            // loaded successfully
-            sound.play(() => {
-              sound.reset();
-            });
+        const sound = new Sound(reminderSound, Sound.MAIN_BUNDLE, (error) => {
+          if (error) {
+            return;
           }
-        );
+          // loaded successfully
+          sound.play(() => {
+            sound.reset();
+          });
+        });
       }
     }
   }
 
   handleOnPress(item, navigator) {
     if (!item.folder) {
-      const { setCurrentShabad} = this.props;
+      const { setCurrentShabad } = this.props;
       setCurrentShabad(item.id);
-      navigator.navigate('Reader', {
+      navigator.navigate("Reader", {
         key: `Reader-${item.id}`,
         params: { item },
       });
     } else {
-      navigator.navigate('FolderBani', {
+      navigator.navigate("FolderBani", {
```

Comment:
It's strongly recommended to put navigation tabs in a common file variable instead of using them directly.

Reply:
fixed